Keas.Mvc/Controllers/Api/DocumentController.cs

```diff
@@ -119,5 +119,38 @@ public async Task<IActionResult> Create([FromBody] Document document)
 
             return Json(newDocument);
         }
+
+        [HttpPost("{id}")]
+        [ProducesResponseType(typeof(Document), StatusCodes.Status200OK)]
+        public async Task<IActionResult> Delete(int id)
+        {
+            var document = await _context.Documents.Where(x => x.Team.Slug == Team)
+                .Include(x => x.Team)
+                .SingleAsync(x => x.Id == id);
+
+            if (document == null)
+            {
+                return NotFound();
+            }
+
+            if (!document.Active)
+            {
+                return BadRequest(ModelState);
+            }
+
+            using (var transaction = _context.Database.BeginTransaction())
+            {
+
+                _context.Documents.Update(document);
```

Comment:
Do you need this line?  Shouldn't it be implicit because you grabbed the object already and didn't do noTracking so it should be tracked already.

Reply:
I copied from equipment.